sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -695,6 +695,12 @@ func withDefaultMounts(containerID string, persistentDir string) oci.SpecOpts {
 			Source:      "cgroup",
 			Options:     []string{"ro", "nosuid", "noexec", "nodev"},
 		},
+		// Bottlerocket release information for the container
```

Comment:
Is this required if it can be accessed from `/.bottlerocket/rootfs/etc/release`?